e2e/run.sh

```diff
@@ -32,7 +32,7 @@ stop_kind() {
 run_tests() {
     (
         cd "$SCRIPT_DIR"
-        telepresence helm upgrade
+        telepresence helm install
```

Comment:
`telepresence helm upgrade` results in an error unless I am missing another `install` command somewhere else. So, I think this should be `telepresence helm install`. 🤔 @charithe

Reply:
If it's an issue with Telepresence, the other tests would have the same problem as well. What I'd do is:

- Undo the `svc.cluster.local` changes in this PR. They shouldn't be necessary.
- Create a branch directly in the cerbos repo with your change to Telepresence. 
- Launch the E2E workflow from that branch and see if it succeeds.